### ISSUE TITLE
physlock doesn't automatically lock, because it lost the -u option

### ISSUE THREAD

**Issue by magnetophon** [NixOS/nixpkgs#21935]
## Issue description

physlock used to have an option to specify the user that is allowed to unlock, but lost that option now.
I did a PR to upgrade physlock (for other reasons), but I failed to update [physlock.nix](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/security/physlock.nix#L108).

I think it's just a matter of getting rid of the ``-u userName``, but I'm not sure, so I'm not doing a PR (yet).

**Comment by dtzWill** [comment]
I'm encountering this issue as well, after switching from 16.09 to unstable.  You say "automatically lock"--can it still be invoked manually somehow? My normal use-case is running 'physlock' before leaving, which doesn't seem to work anymore.

journalctl contains the following:

```
Feb 22 18:02:51 trypticon physlock-start[21252]: /nix/store/2qqn8dn9bgr73j7ngalkgppfbv8nca73-physlock-v0.5/bin/physlock: invalid option -- 'u'
Feb 22 18:02:51 trypticon physlock-start[21252]: usage: physlock [-dhLlmsv]
Feb 22 18:02:51 trypticon systemd[1]: physlock.service: Control process exited, code=exited status=1
Feb 22 18:02:51 trypticon systemd[1]: Failed to start Physlock.
```

**Comment by magnetophon** [comment]
I think [this](https://github.com/NixOS/nixpkgs/pull/23762) is a fix.

**Comment by globin** [comment]
Fixed in #23762
